sssd_test_framework/utils/authentication.py

```diff
@@ -340,7 +340,17 @@ def password_expired(self, username: str, password: str, new_password: str) -> b
 
         return result.rc == 0
 
-    def passkey(self, username: str, *, pin: str | int, device: str, ioctl: str, script: str) -> bool:
+    def passkey(
+        self,
+        username: str,
+        *,
+        pin: str | int,
+        device: str,
+        ioctl: str,
+        script: str,
+        command: str | Any,
+        expected_output_from_cmd: str | Any
```

Comment:
Is "Any" better here than "None = ''"?

Like we discussed earlier, the issue you found when `command` is defined and `expected_output_from_cmd` is set to an empty string, may require special handling.   Just something to consider.

Reply:
updated the code.